docker/quorum-k8s-hooks/Dockerfile

```diff
@@ -2,21 +2,18 @@ FROM debian:latest
 
 RUN set -eux; \
     mkdir -p /root/.kube/ ; \
-    #https://github.com/balena-io-library/base-images/issues/273
-    mkdir -p /usr/share/man/man1; \
     apt-get update && apt-get install -y --no-install-recommends \
-    pkg-config ca-certificates apt-transport-https gnupg gnupg2 lsb-release software-properties-common jq libc6-dev make curl wget vim unzip libsodium-dev; \
+    pkg-config ca-certificates apt-transport-https gnupg gnupg2 lsb-release software-properties-common jq libc6-dev make curl wget unzip libsodium-dev; \
```

Comment:
I'd like to keep vim in here to make debugging of things easier - its also not a heavyweight package

Reply:
got it, totally makes sense.